paimon-core/src/main/java/org/apache/paimon/operation/OrphanFilesClean.java

```diff
@@ -91,6 +95,10 @@ public abstract class OrphanFilesClean implements Serializable {
     protected final int partitionKeysNum;
     protected final Path location;
 
+    private static final String THREAD_NAME = "ORPHAN-FILES-CLEAN-THREAD-POOL";
+    private static final ThreadPoolExecutor executorService =
```

Comment:
Do not introduce another thread pool, for `LocalOrphanFilesClean`, you should just use executor in it.